ext/hash/tests/hash_update_file.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+bool hash_update_file ( resource $hcontext , string $filename [, resource $scontext = NULL ] );
+--CREDITS--
+marcosptf - <marcosptf@yahoo.com.br> - @phpsp - sao paulo - br
+--SKIPIF--
+<?php
+if (phpversion() < "5.3.0") {
```

Comment:
We do not need this, PHP 5.3 is long gone and not supported and we generally tend not to have version checks in the source. If a test do not work on a certain version (such as a new function added in x but not y), then we only commit it to x's branch and newer